Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM public.ecr.aws/unocha/python:3
+
+WORKDIR /srv/hapi
+
+COPY . .
+
+RUN apk add unit unit-python3 && \
+    apk --virtual .build-deps add \
+        git \
+        build-base git python3-dev postgresql-dev && \
+    mkdir -p \
+        /etc/services.d/hapi \
+        /srv/cache \
```

Comment:
do you actually use these folders? if you do, let them in. if not, remove them
it looks like hxl-proxy to me.